### DISCUSSION TITLE
`GLIBC_2.29' not found Error

### DISCUSSION THREAD

**Discussion by mudgen** [pawelsalawa/sqlitestudio#4449] [General]
### Details
I tried installing the version of sqlitestudio on this webpage: https://github.com/pawelsalawa/sqlitestudio/actions/runs/2329572174
I am using MX Linux.

When I execute `sqlitestudio` I get this error:
`./sqlitestudio: /lib/x86_64-linux-gnu/libm.so.6: version 'GLIBC_2.29' not found (required by /home/mudgen/Desktop/sqlite/SQLiteStudio/lib/libsqlite3.so)`

Is there a way I can fix this and get it working?

**Comment by pawelsalawa** [comment]
It means your linux has different GLIBC version than the one in Ubuntu, that is used by GitHub actions.
I'm not aware of quick solution. The long solution is to build the binary yourself, on your system, with your glibc.

**Comment by mudgen** [comment]
Understood, thanks for the answer.
